src/main/java/com/emc/ecs/servicebroker/service/EcsService.java

```diff
@@ -138,8 +146,17 @@ public CompletableFuture wipeAndDeleteBucket(String bucketName, String namespace
 
             logger.info("Started wipe of bucket '{}' in namespace '{}'", bucketName, namespace);
             BucketWipeResult result = bucketWipeFactory.newBucketWipeResult();
-            bucketWipe.deleteAllObjects(prefix(bucketName), "", result);
 
+            if (isBucketVersioningEnabled(prefix(bucketName))) {
+                bucketWipe.deleteAllVersions(prefix(bucketName), "", result);
+                logger.info("Deleted all versions of bucket '{}' in namespace '{}'", bucketName, namespace);
```

Comment:
I would be consistent with these logs - IMO, both logs should write _**before**_ the operation.

Also, I would use the prefixed bucket name in the log, as we saw when troubleshooting, it was hard to know from the log whether the actual bucket name was being prefixed in the calls

Reply:
Updated the  log statements to include prefixed bucket name